### PR TITLE
Witcher.js improvements

### PR DESCRIPTION
**Pull request by mclemente**
I've been making some improvements on the project, but I'll wait to see what you're working on the dev branch, but here's some for the witcher.js file.

Noteworthy changes:
- meleeBonus can be calculated as `((BODY-6)/2)*2` instead of a switch.
- STUN can be clamped in a value between 1 and 10.

### CODE REVIEW

File: module/witcher.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export function getRandomInt(max) {
-    return Math.floor(Math.random() * (max + 1)) + 1;
+		return Math.floor(Math.random() * (max + 1)) + 1;
```
Comment (Batziy) [inline module/witcher.js:2]: Feels like a lot of unnecessary "tabs" :P

Comment (AnthonyMonette) [reply module/witcher.js:2]: Yeah you might need to change the configs of your IDE, it's seems to put a lot of uncessary tabs

Comment (mclemente) [reply module/witcher.js]: Oops, I've replaced two whitespaces instead of four. Should be good now.

File: module/witcher.js [hunk 2]
```diff
@@ -11,114 +11,30 @@ with the Hand to Hand Table, page 48 of Witcher TRPG Handbook.
...
-	let baseMax = Math.floor((thisActor.data.data.stats.body.max + thisActor.data.data.stats.will.max)/2);
-    let currentBody = thisActor.data.data.stats.body.current;
+	if (actor.data.data.customStat === true){
+		return null;
+	}
+	const thisActor = actor;
+	const stats = thisActor.data.data.stats;
+	const base = Math.floor((stats.body.current + stats.will.current)/2);
+	const baseMax = Math.floor((stats.body.max + stats.will.max)/2);
+	const meleeBonus = Math.ceil((stats.body.current-6)/2)*2;
```
Comment (AnthonyMonette) [inline module/witcher.js:21]: Nice!

Comment (AnthonyMonette) [reply module/witcher.js:21]: It still seems to have a lot of uncessary spacing. Il try to create a config file that we can all use.

**Comment by AnthonyMonette** [comment]
You should merge in dev not in main

**Pull request merged**